### DISCUSSION TITLE
Can you please keep keepassxc from bringing in personal political opinions?

### DISCUSSION THREAD

**Discussion by yahahaaaaa** [keepassxreboot/keepassxc#10566] [Q&A]
Chinese is divided into Chinese (China) and Chinese (Taiwan) in the language settings. Can it be changed to Chinese (Simplified) and Chinese (Traditional)? This is a free and open source software. I don't think it's appropriate to express personal political opinions like this. It will only cause unnecessary controversy. If you like to cause controversy, use your personal identity to express your opinions. This looks like a good software, it's my first time using it and I don't want to give up using it for this reason

**Comment by phoerious** [comment]
The IETF language code is zh_TW. Please direct your anger to them.

**Reply by yahahaaaaa** [reply]
I'm not here to get angry, just a suggestion. Actually I'm not angry either. I found it strange because I used keepass first and its translation was in (simplified)(traditional), but keepassxc was not. Thank you for your answer.

**Comment by droidmonkey** [comment]
> It will only cause unnecessary controversy

That is what you are doing, not us.

**Reply by yahahaaaaa** [reply]
I am in no way trying to cause controversy, quite the opposite. Taiwan has special historical meaning to China, and I know that most Chinese people are uncomfortable with this naming method. The naming methods of Chinese (Simplified) and Chinese (Traditional) are more neutral, in my opinion. But that’s not the point. I don’t care at all if I see political comments on Twitter or YouTube. I hate having to argue with people about this and that when using open source software. I just don't feel like it's appropriate here. However, you may feel that the naming method of Chinese (Simplified) (Traditional) has political overtones, well, if so, this is a cognitive difference. (Can you please delete my post after reading this? If so, there is really no point in arguing about these, it’s too stupid)

**Reply by droidmonkey** [reply]
I am sorry to say that when you title your discussion, "please keep your personal political opinions," then you start the discussion with that framing. We don't have political opinions expressed through KeePassXC, these are translation languages that are sourced from transifex.com with many participants. I wasn't even aware of a 'controversy' in language choices.

**Reply by yahahaaaaa** [reply]
If that's the case, I apologize to you
